### PR TITLE
images bootstrap

### PR DESCRIPTION
**Pull request by sergioramos**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,36 @@
 # webconsole-images
-Images webconsole 
+
+[![License: MPL 2.0](https://img.shields.io/badge/License-MPL%202.0-brightgreen.svg)](https://opensource.org/licenses/MPL-2.0) [![Build Status](https://secure.travis-ci.org/joyent/webconsole-images.svg)](http://travis-ci.org/joyent/webconsole-images)
+
+Web console for instance management.
```
Comment (cjihrig) [inline README.md]: s/instance/image/ ?

File: test/index.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+const Code = require('code');
+const Lab = require('lab');
+
+// Test shortcuts
+const lab = exports.lab = Lab.script();
+const { describe, it } = lab;
+const { expect, fail } = Code;
+
+
+describe('Instance Console', () => {
```
Comment (cjihrig) [inline test/index.js]: Image Console?

**Review by cjihrig — APPROVED** [review]

**Comment by geek** [comment]
LGTM with @cjihrig's suggestions

**Comment by sergioramos** [comment]
updated

**Pull request merged**